routers/likes.py

```
@@ -1,31 +1,143 @@
 
-# register like
 @router.post("/posts/{post_id}/likes")
-async def post_like(post_id: PostId):
-    return {"success": "post_like"}
+def create_like(post_id: PostId, user_id: CurrentUserId):
+    """좋아요 등록"""
+    _, posts = _get_post_or_404(post_id)
+    likes = read_json(settings.likes_file)
+
+    # 이미 좋아요한 경우 확인
+    if any(l["post_id"] == post_id and l["user_id"] == user_id for l in likes):
```

Ruff (0.14.13)

Ambiguous variable name: `l`

(E741)